cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/discover.spec.js

```diff
@@ -124,13 +113,6 @@ describe('discover app', { scrollBehavior: false }, () => {
         .should('have.text', saveSearch2);
     });
 
-    it('should show the correct hit count', function () {
-      cy.loadSaveSearch(saveSearch2);
-      cy.setTopNavDate(DE_DEFAULT_START_TIME, DE_DEFAULT_END_TIME);
-      const expectedHitCount = '14,004';
```

Comment:
we should figure out what happened here as it was consistent before. perhaps someone modified the sample data?